WearOAuth/oauth-pkce/src/main/java/com/example/android/wearable/oauth/pkce/AuthPKCEViewModel.kt

```diff
@@ -78,16 +85,16 @@ class AuthPKCEViewModel(application: Application) : AndroidViewModel(application
                 .encodedPath("https://accounts.google.com/o/oauth2/v2/auth")
                 .appendQueryParameter("scope", "https://www.googleapis.com/auth/userinfo.profile")
                 .build()
-            val oauthRequest = OAuthRequest.Builder(getApplication())
+            val oauthRequest = OAuthRequest.Builder(context)
```

Comment:
nice

Reply:
Although I do wonder if this is racy? Could the activity be replaced or stopped, by the time the view model scope gets to running this?

Should this part be done outside the launch.

This is a nit, but shows why mixing activity context and viewmodel is hard to reason about.

Reply:
Agreed that this may be racy.  Looking into this in the next pull request.